### ISSUE TITLE
Changing tabs on a prefix-less ASN throws a JS error

### ISSUE THREAD

**Issue by linuxgemini** [bgptools/issues#2]
URL:
```
https://bgp.tools/as/213230#whois
```

Browser (User-Agent if you can): 
```
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.61 Safari/537.36
```

What were you expecting:
```
Whois information of AS213230
```

What happened (JS Console output):
```
Uncaught TypeError: Cannot read property 'children' of null
    at hideAllContent (213230:140)
    at HTMLAnchorElement.<anonymous> (213230:161)
```

**Comment by linuxgemini** [comment]
Looks like the JS code for `hideAllContent()` in-site has a code to modify the styles of `prefixlist-tbody` on the page, however this ASN does not have any prefixes. Thus no `prefixes-page` and no error handling on this makes it break.

![in-site JS snippet](https://btw.i-use-ar.ch/i/rszve8b1.png)

**Comment by benjojo** [comment]
Thanks for the report, Will look into this tonight

**Comment by benjojo** [comment]
Fixed in 7e616f4
